backend/de.metas.business/src/main/sql/postgresql/system/20-de.metas.business/5671210_sys_gh14346_customize_material_receipt_window.sql

```diff
@@ -0,0 +1,11 @@
+-- Field: Wareneingang_OLD -> Wareneingang -> Referenz
+-- Column: M_InOut.POReference
+-- 2023-01-11T10:57:13.070Z
+UPDATE AD_Field SET AD_Reference_ID=10,Updated=TO_TIMESTAMP('2023-01-11 11:57:13.069','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Field_ID=3502
```

Comment:
why is this needed?

Reply:
Align  it with the order display. one line display